packages/shared-data/pricing.ts

```diff
@@ -509,12 +509,12 @@ export const pricing: Pricing = {
         usage_based: false,
       },
       {
-        title: 'Log drain',
+        title: 'Log drains',
         plans: {
           free: false,
           pro: false,
-          team: 'Coming soon',
-          enterprise: 'Coming soon',
+          team: '$50 per destination per project per month',
```

Comment:
Pricing page changes need to start in Notion and be approved by Rory before making any changes in the frontend.

New iteration https://www.notion.so/supabase/Pricing-v7-7-412d0cf8b2544ad7a73cfacdc827a13e?pvs=4

I've added some questions